### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by peanutman1116** [Najmul190/Discord-AI-Selfbot#29]
File "C:\Users\yourname\Downloads\Discord-AI-Selfbot-main\Discord-AI-Selfbot-main\main.py", line 22, in <module>
    selfbot_id = int(os.getenv("SELFBOT_ID"))
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

**Comment by Najmul190** [comment]
did you put in your selfbot id in your .env file?

**Comment by peanutman1116** [comment]
Yes I did. Just to verify it would be the user ID of the selfbot right?

**Comment by peanutman1116** [comment]
> did you put in your selfbot id in your .env file?

Yes I did. Just to verify it would be the user ID of the selfbot right?

**Comment by Najmul190** [comment]
did you rename example.env to just .env

**Comment by peanutman1116** [comment]
> did you rename example.env to just .env

yes still got the error if you want add my discord it would be alot easier to go through trouble shooting discord _error1

**Comment by peanutman1116** [comment]
> did you rename example.env to just .env

Not sure what could be causing the problem now

**Comment by R1Gel2** [comment]
same issue here with the exact same error

**Comment by UserGG0** [comment]
ye same

**Comment by pnydiffuser** [comment]
> ye same

there will be a huge update to fix bugs eventually. The developer responds but is very busy making it up to date.
